bin/cleanInstall.js

```diff
@@ -97,7 +97,12 @@ function deleteFolderRecursive(path) {
 }
 function install(packages, callback) {
     console.log('installing ', packages);
-    const child = exec('npm prune; npm install; npm install ' + packages + ' --save -E; mvn clean install -DskipTests',
+    let command = 'npm prune; npm install; npm install ' + packages + ' --save -E;';
```

Comment:
I'd have though you'd want this to bomb if any of those failed? i.e. separate the commands with `&&` ?